hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/DeltaSync.java

```diff
@@ -236,6 +239,8 @@ public class DeltaSync implements Serializable, Closeable {
    */
   private transient SparkRDDWriteClient writeClient;
 
+  private Option<BaseQuarantineTableWriter> quarantineTableWriterInterfaceImpl = Option.empty();
+
```

Comment:
can we call this "quarantineTableWriter"

Reply:
Addressed in latest commit.